src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/FrameOfT.cs

```diff
@@ -188,6 +199,15 @@ public class Frame<TContext> : Frame
                 // SetBadRequestState logs the error.
                 SetBadRequestState(ex);
             }
+            catch (IOException ex) when (ex.InnerException is UvException)
+            {
+                // Don't log ECONNRESET errors made between requests. Browsers like IE will reset connections regularly.
+                if (requestLineStatus != RequestLineStatus.Empty ||
+                    ((UvException)ex.InnerException).StatusCode != Constants.ECONNRESET.Value)
+                {
+                    Log.RequestProcessingError(ConnectionId, ex);
+                }
```

Comment:
We already log that in `Connection.OnRead`. Part of this was getting rid of that double logging.